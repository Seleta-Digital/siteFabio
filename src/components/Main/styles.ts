import styled, { css } from 'styled-components'

export const Wrapper = styled.main`
  ${({ theme }) => css`
    color: ${theme.colors.mainTxt};
    width: 100vw;
    height: 100vh;
  `}
`
export const Descriptions = styled.p`
  ${({ theme }) => css`
    font-size: 1.2rem;
    color: ${theme.colors.mainTxt};
  `}
`
