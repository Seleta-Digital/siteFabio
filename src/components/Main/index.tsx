import * as S from './styles'
const Main = () => (
  <S.Wrapper>
    <h1>Fabio Morelli Site em construção</h1>
  </S.Wrapper>
)

export default Main
