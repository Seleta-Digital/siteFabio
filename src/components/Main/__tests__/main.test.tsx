import { render, screen } from '@testing-library/react'

import Main from '../'

describe('<Main />', () => {
  it('should render heading', () => {
    render(<Main />)
    expect(screen.getByRole('heading', { level: 1 }))
  })
  it('should render background red', () => {
    const { container } = render(<Main />)
    expect(container.firstChild).toHaveStyle({ 'background-color': 'red' })
    expect(container.firstChild).toMatchSnapshot()
  })
})
