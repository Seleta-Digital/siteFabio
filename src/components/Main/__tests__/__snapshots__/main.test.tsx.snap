// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should render heading 1`] = `
.c0 {
  background-color: #06092b;
  color: #fff;
}

<main
  class="c0"
>
  <h1>
    Fabio Morelli Site em construção
  </h1>
</main>
`
